# Polygon mesh class
# Based on http://docs.pointclouds.org/trunk/structpcl_1_1_polygon_mesh.html

Header header
vrteleop/HeaderlessPointCloud2 cloud
vrteleop/Polygon[] polygons
